swan-lake/development-tutorials/deployment-guide/virtual-machine-deployment.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: ballerina-cloud-left-nav-pages-swanlake
+title: Virtual machine deployment
+description: If you prefer to execute Ballerina services directly on virtual machines (VMs) or bare metal servers without using containerization, you can follow these steps to set up and run your services in a production environment, 
+keywords: ballerina, programming language, services, cloud, kubernetes, docker
+active: ballerina-deployment-guideLines
+intro: If you prefer to execute Ballerina services directly on virtual machines (VMs) or bare metal servers without using containerization, you can follow these steps to set up and run your services in a production environment, 
+---
+
+1. Prepare the Server Environment.
+    - Provision the VM or Bare Metal Server,
```

Comment:
Why we need two level of bullet points? It can be follows?

```
1. Prepare the Server Environment by provisioning the VM or Bare Metal Server.
    - Ensure the server meets the hardware requirements for your application (CPU, memory, disk space, etc.).
    - Configure the server OS (Linux is recommended for production).
```